packages/lesswrong/components/common/LWHome.tsx

```diff
@@ -33,8 +33,7 @@ const LWHome = () => {
             <FrontpageReviewWidget reviewYear={REVIEW_YEAR}/>
           </SingleColumnSection>}
           <SingleColumnSection>
-            {/* change this to "current" after Petrov Day period */}
-            <DismissibleSpotlightItem spotlight={document}/>
+            <DismissibleSpotlightItem current/>
```

Comment:
Please also remove the `useSingle` which was fetching the `document` (hardcoded spotlight)